### DISCUSSION TITLE
not showing hic track

### DISCUSSION THREAD

**Discussion by miaoxm2** [GMOD/jbrowse-components#3736] [Q&A]
Hi, 
I am using JBrowse 2 on Mac trying visualize the HiC contact map. 
The .hic file was generated by Juicer, and it works well using juicebox. I found some other discussion mentioned the high and low letter of seq ID. So I've changed the original fasta and fai file to match the ones showing in hic(from juicebox, all capital letters).
Still doesn't work.
Would you please help me through this?
Thanks,
Xiaomeng
<img width="1012" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/108287599/8231525b-b32e-4267-8698-346a127ac77f">
<img width="215" alt="image" src="https://github.com/GMOD/jbrowse-components/assets/108287599/b8f912c5-305f-4a82-b146-f05087a3bf86">

**Comment by cmdcolin** [comment]
Did you manage to get it working? Just saw this issue was closed
